03.create/04.entities/14.primitive-painting-set-tutorial/docs.md

```diff
@@ -0,0 +1,120 @@
+- [Write Your Own Scripts](../../../script/write-scripts)
+
+## Create a Painting Set
+
+Our painting set comprises three elements:
+* [A brush, made of a cylinder and a sphere object](#create-a-paint-brush).
+* [A palette, made of an octagon and sphere object](#create-a-paint-palette).
+* [A canvas, made of box entities](#create-a-pixel-canvas).
+
+All of the logic for our painting set is contained in the brush head. The rest of the content is made by parenting primitives to one another to make our brush, palette, and canvas.
+
+### Create a Paint Brush
+We'll start by creating the paint brush. The brush is comprised of two parts, the handle and the brush head. The brush handle is the parent of the brush head. This means we can grab and move the brush, and select the color to be painted using the handle only. 
+
+To create the brush handle:
+1. In Interface, pull up your HUD or Tablet and go to **Create**.
+2. [Create an entity](../create-entities) to be used as the handle. This can be a box or sphere entity.
+3. Go to the 'Properties' tab and select the 'Shape' drop down. Change the shape of the entity to a 'cylinder'.
+4. Name your entity 'Paint-Paintbrush-Tube' by selecting the text box at the top of the 'Properties' tab.
+5. Select your desired handle color from the 'Color' picker.
+6. Scroll down to 'Spatial'. Change the local dimensions to: {x: 0.025, y: 0.5, z: 0.025}.
+
+To create the brush head:
+1. In Interface, pull up your HUD or Tablet and go to **Create**.
+2. Click on the ['SPHERE' icon](../create-entities) to create a sphere entity to be used as the brush head.
+3. Go to the 'Properties' tab and select your desired brush head color from the 'Color' picker.
+4. Name your entity 'Paint-Paintbrush-Head' by selecting the text box at the top of the 'Properties' tab.
+5. Scroll down to 'Spatial'. Change the local dimensions to {x: 0.05, y: 0.1, z: 0.05}.
+
+Once you've created the brush head, you can parent the brush handle to it: 
+
+1. In **Create** Tools app, select your brush handle and go to the 'Properties' tab.
+2. Copy the 'entityID'.
+3. From the Entity List window, select the brush head and go to the 'Properties' tab.
+4. In the 'Parent' text box, paste the entityID of the brush handle (Paintbrush-Tube) entity.
+5. Scroll down to 'Spatial'. Change the local position to {x: 0, y: 0.2, z: 0}.
+
+We've detailed how you can add a script to use the brush to transfer color to other objects in [Add a Paint Brush Script](#add-a-paint-brush-script).
+
+### Create a Paint Palette
+The second part of our painting set is the palette. This is where you can get creative, and add as many (or as few) paint colors as you'd like. Once you have the base of the palette created, you can repeat the process of adding paints until you are satisfied with the end result.
+
+To create the palette base:
+1. From the **Create** Tools app, [create an entity](../create-entities) to be used as the palette base. This can be a box or sphere entity.
```

Comment:
```suggestion
1. In Interface, pull up your HUD or Tablet and go to **Create**.
2. [Create an entity](../create-entities) to be used as the palette base. This can be a box or sphere entity.
```

Reply:
done